dvc/scm/git.py

```diff
@@ -145,6 +147,116 @@ def branch(self, branch):
     def tag(self, tag):
         self.git.git.tag(tag)
 
+    def open(self, path, branch=None):
+        if not branch:
+            return open(path)
+
+        relpath = os.path.relpath(path, self.root_dir)
+
+        # XXX: make git_object_by_path working with relative paths?
```

Comment:
I'm not sure I understand this XXX. Could you clarify it a little bit or resolve?